### DISCUSSION TITLE
.NET 4.6 project cannot add CSLA via NuGet (Microsoft.Extensions.Configuration 3.1.9)

### DISCUSSION THREAD

**Discussion by michaelcsikos** [MarimerLLC/csla#2275] [Questions]
Revit 2018 and AutoCAD 2018 both target .NET 4.6 (not 4.6.1). When trying to add CSLA via NuGet to a new .NET 4.6 project it fails with the error:

`Could not install package 'Microsoft.Extensions.Configuration 3.1.9'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.6', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.`

Am I doing something wrong?

In the past we have manually set up references, and in this case would use the .NET 4.5 library, but we're hoping to be able to use NuGet moving forward.

**Comment by rockfordlhotka** [comment]
What version of CSLA?

**Reply by michaelcsikos** [reply]
5.4.2. It's very easy to reproduce, just create a new class library targeting .NET Framework 4.6 and try to install CSLA via NuGet.

**Reply by michaelcsikos** [reply]
Microsoft.Extensions.Configuration.3.1.9 only has libraries for `netcoreapp3.1` and `netstandard2.0` which was released in .NET 4.6.1, of course. In my opinion this was too significant for such a small version bump; Microsoft should have called this 4.7.
